deps/exokit-bindings/glfw/src/glfw.cc

```diff
@@ -1351,8 +1351,15 @@ NAN_METHOD(GetClipboard) {
 
 NAN_METHOD(SetClipboard) {
   NATIVEwindow *window = (NATIVEwindow *)arrayToPointer(Local<Array>::Cast(info[0]));
-  String::Utf8Value str(info[0]->ToString());
-  glfwSetClipboardString(window, *str);
+  if (info.Length() >= 2) {
+    Nan::Utf8String utf8_value(info[1]);
+    int len = utf8_value.length();
+    std::string name(*utf8_value, len);
```

Comment:
What is this line for?

Reply:
Im not sure TBH, it worked so i didn't change it from what it was:

would this work better:

```
NAN_METHOD(SetClipboard) {
  if (info.Length() >= 2) {
    NATIVEwindow *window = (NATIVEwindow *)arrayToPointer(Local<Array>::Cast(info[0]));
    String::Utf8Value str(info[1]->ToString());
    glfwSetClipboardString(window, str);   
  } else {
      return Nan::ThrowTypeError(
            "must provide arg length > 1");
  }
}
```

Reply:
oh yes duh, i see what you mean now. I'm a doofus. Ok how does the latest look?